src/DynamoCore/Graph/Nodes/NodeModel.cs

```diff
@@ -1617,6 +1631,18 @@ public virtual void ClearErrorsAndWarnings()
             OnNodeMessagesClearing();
         }
 
+        /// <summary>
+        /// Clears the info messages that are generated when running the graph,
+        /// the State will be set to ElementState.Dead.
+        /// </summary>
+        public virtual void ClearInfoMessages()
+        {
+            infos.RemoveWhere(x => x.State == ElementState.Info);
+            State = ElementState.Dead;
+            ClearTooltipText();
```

Comment:
I think this method will be replaced by `ClearTransientWarningsAndErrors` from @aparajit-pratap 's [PR](https://github.com/DynamoDS/Dynamo/pull/12712)

Reply:
In the future, if a particular node has any warnings/errors and info messages on it, we don't want to clear the warnings/errors incase we want to just remove the info message. That's why I created a separate method for that use.

Reply:
Yup, I understand that, I am referring to `ClearTooltipText ` here, that it is being removed

Reply:
Call ClearTransientWarningsAndErrors instead of ClearTooltipText?

Reply:
Yes, but after that PR gets merged.

Reply:
ClearTransientWarningsAndErrors would remove the errors and warnings from infos list, which is not needed.

Reply:
@reddyashish `ClearToolTipText` has been renamed as `ClearTransientWarningsAndErrors`

Reply:
> ClearTransientWarningsAndErrors would remove the errors and warnings from infos list, which is not needed.

`ClearToolTipText` is executing the same code as `ClearTransientWarningsAndErrors`
Only the name has been updated, as TooltipText property is removed now, so I suggest that we use `ClearTransientWarningsAndErrors` here.

Reply:
@zeusongit I think @reddyashish intends to _only_ clear info messages in this method, not other errors and warnings. If that's the case, `ClearTransientWarningsAndErrors` (`ClearToolTipText`) should _not_ be called.

Reply:
Also, why is the node's state being set to `ElementState.Dead`?

Reply:
Yes, in that case warnings and errors should not be removed, I think the misleading method name `ClearToolTipText ` may have created this confusion.

Reply:
Removed the ClearToolTipText() to avoid confusion.

Reply:
@aparajit-pratap Removed the statement where it is being set to ElementState.Dead.